src/middleware/joi-validator.ts

```diff
@@ -1,26 +1,33 @@
 import * as Joi from "joi"
 import Boom = require("boom");
 import Extend = require("extend");
+import { NextFunction, Response } from "express";
 
-export default function validate(schema: any, options: any = {}) {
+export default function validate(schema: Joi.SchemaLike, options: Joi.ValidationOptions = {}): (req: Request, res: Response, next: NextFunction) => void {
     options.abortEarly = false;
 
     // Using Request from express I get typescript errors
-    return function validateRequest(req: any, res: Response, next: any) {
-        let toValidate: any = {};
+    return function validateRequest(req: Request, res: Response, next: NextFunction): void {
+        // This is a generic callback which can be used for any object, thus this can be any as well
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const toValidate: any = {};
         if (!schema) {
             next();
```

Comment:
Should this be `return`ed?

Reply:
Good call, doesn't break but it probably can cause an issue